lib/default/TasmotaSerial-3.3.0/src/TasmotaSerial.cpp

```diff
@@ -29,6 +29,11 @@ extern "C" {
 
 #ifdef ESP8266
 
+// added to access kTasmotaSerialConfig
+#include "../../../../tasmota/tasmota_compat.h"
+#include "../../../../tasmota/tasmota.h"
```

Comment:
Use extern definition instead.

Reply:
Apparently you cannot extern a const since it's not a variable so the linker cannot find it.

I moved kTasmotaSerialConfig from Tasmota.h to TasmotaSerial.h to address this.